src/app/inventory/ItemPowerSet.m.scss

```diff
@@ -11,7 +11,7 @@
   img {
     height: 20px;
     width: 20px;
-    &.invert {
+    &.invert:not(:global(.colorized)) {
```

Comment:
Would it be possible to just use `!important` to force the filter off w/ `colorized`?